code/game/objects/items/toys.dm

```diff
@@ -1089,6 +1089,25 @@
 	else
 		to_chat(user, "<span class='alert'>Nothing happens.</span>")
 
+/*
+ * Toy  train
+ */
+/obj/item/toy/toytrain
+	name = "toy train"
+	desc = "A little toy train for young spess men. The wheels are attached to a piece of string!"
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "toytrain"
+	w_class = WEIGHT_CLASS_SMALL
+	var/cooldown = 0
+
+/obj/item/toy/toytrain/attack_self(mob/user)
+	if (cooldown < world.time)
+		cooldown = (world.time + 20) // Sets cooldown at 2 seconds
+		user.visible_message("<span class='notice'>[user] pulls the string on the bottom of the train.</span>", "<span class='notice'>You pull the string, the wheels begin to spin.</span>", "<span class='italics'>The train whires audibly.</span>")
+		playsound(src, 'sound/voice/moth/scream_moth.ogg', 5, 0)
```

Comment:
I don't remember what this sounds like but it's called `scream_moth` so it doesn't seem appropriate for a toy train.

Reply:
The noise is supposed to be the the wheels whirring after the string is pulled, `scream_moth` is the best I could find for it. It sounds similar to the bee plush and I think it is the sound the moth plush uses if you can remember those.